app/org/sagebionetworks/bridge/hibernate/HibernateAccountDao.java

```diff
@@ -276,13 +313,16 @@ public void updateAccount(Account account) {
         String accountId = account.getId();
         HibernateAccount accountToUpdate = marshallAccount(account);
 
-        // Can't change study, email, createdOn, or passwordModifiedOn.
+        // Can't change study, email, phone, emailVerified, phoneVerified, createdOn, or passwordModifiedOn.
         HibernateAccount persistedAccount = hibernateHelper.getById(HibernateAccount.class, accountId);
         if (persistedAccount == null) {
             throw new EntityNotFoundException(Account.class, "Account " + accountId + " not found");
         }
         accountToUpdate.setStudyId(persistedAccount.getStudyId());
         accountToUpdate.setEmail(persistedAccount.getEmail());
+        accountToUpdate.setPhone(persistedAccount.getPhone());
+        accountToUpdate.setEmailVerified(persistedAccount.getEmailVerified());
+        accountToUpdate.setPhoneVerified(persistedAccount.getPhoneVerified());
```

Comment:
Update HibernateAccountDaoTest.updateSuccess() to make sure we're preventing updates to phone and the verified flags?